tests/core/pyspec/eth2spec/test/deneb/sanity/test_blocks.py

```diff
@@ -16,32 +18,134 @@
 )
 
 
+def run_block_with_blobs(spec, state, blob_count):
+    yield 'pre', state
+
+    block = build_empty_block_for_next_slot(spec, state)
+    opaque_tx, _, blob_kzg_commitments, _ = get_sample_opaque_tx(spec, blob_count=blob_count)
+    block.body.blob_kzg_commitments = blob_kzg_commitments
+    block.body.execution_payload.transactions = [opaque_tx]
+    block.body.execution_payload.excess_data_gas = 2
```

Comment:
Just to understand, here we are testing that any non-zero `excess_data_gas` value will be accepted?

Reply:
@asn-d6 correct.